packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -41,14 +61,14 @@ improved accessibility.
 
 ## Placement
 
-Tooltip implements the [Placement Dictionary][dictionary-placement] for placement. The dictionary values are used as CSS
-modifiers in the camelCase format: `Tooltip--top`, `Tooltip--rightTop`, `Tooltip--leftBottom`, etc.
+Tooltip implements the [Placement Dictionary][dictionary-placement] for placement. The dictionary values are used as
+a value of data attribute `data-spirit-placement`, e.g. `data-spirit-placement="top"`, `data-spirit-placement="rightEnd"`, etc.
 
-For JS-controlled positioning please use the `data-spirit-placement` attribute instead of CSS modifiers (more on that
-[below](#advanced-positioning)).
+For JS-controlled positioning please use the `data-spirit-placement-controlled` attribute instead of
```

Comment:
```suggestion
For JS-controlled positioning please add the `data-spirit-placement-controlled` attribute instead of specifying the placement using the
```

I'm not sure if it's clear that adding just the attribute itself is fine.